ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -709,16 +1243,26 @@ def checkConvergence(self, traj, new, old):
       @ Out, any(convs.values()), bool, True of any of the convergence criteria was reached
       @ Out, convs, dict, on the form convs[conv] = bool, where conv is in self._convergenceCriteria
     """
-    convs = {}
-    for conv in self._convergenceCriteria:
-      fName = conv[:1].upper() + conv[1:]
-      # get function from lookup
-      f = getattr(self, f'_checkConv{fName}')
-      # check convergence function
-      okay = f(traj, new=new, old=old)
-      # store and update
-      convs[conv] = okay
-
+    if len(self._objectiveVar) == 1:
+      convs = {}
+      for conv in self._convergenceCriteria:
+        fName = conv[:1].upper() + conv[1:]
+        # get function from lookup
+        f = getattr(self, f'_checkConv{fName}')
+        # check convergence function
+        okay = f(traj, new=new, old=old)
+        # store and update
+        convs[conv] = okay
+    else:
+      convs = {}
+      for conv in self._convergenceCriteria:
+        fName = conv[:1].upper() + conv[1:]
+        # get function from lookup
+        f = getattr(self, f'_checkConv{fName}')
+        # check convergence function
+        okay = f(traj, new=new, old=old)
+        # store and update
+        convs[conv] = okay
```

Comment:
I do not see the different between `if` and `else` statements here. If so, I think you need to revert the changes.